cmd/csi-attacher/main.go

```diff
@@ -66,7 +66,7 @@ var (
 	retryIntervalMax   = flag.Duration("retry-interval-max", 5*time.Minute, "Maximum retry interval of failed create volume or deletion.")
 
 	enableLeaderElection    = flag.Bool("leader-election", false, "Enable leader election.")
-	leaderElectionType      = flag.String("leader-election-type", "configmap", "the type of leader election, options are 'configmaps' (default) or 'leases' (recommended). The 'configmaps' option is deprecated in favor of 'leases'.")
+	leaderElectionType      = flag.String("leader-election-type", "configmaps", "the type of leader election, options are 'configmaps' (default) or 'leases' (recommended). The 'configmaps' option is deprecated in favor of 'leases'.")
```

Comment:
Can we define constants for these? It could help avoid typos in the future

Reply:
Good suggestion, already updated, PTAL :)